content/docs/guides/branch-archiving.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Branch archiving
+subtitle: Learn how Neon automatically archives inactive branches to cost-effective storage
+enableTableOfContents: true
+updatedOn: '2024-10-15T17:19:50.116Z'
+---
+
+<Admonition type="note" title="Only enabled on the Free Plan">
+Automatic archiving of inactive branches is only enabled on the Free Plan. Branch archiving will be introduced on paid plans at a later date.
+</Admonition>
+
+To minimize storage costs, Neon automatically archives branches that are:
+
+- Older than **14 days**.
+- Have not been accessed for the past **24 hours**
```

Comment:
maybe include "not marked as protected"

Reply:
Thank you. I'll add this when we update the doc for release to paid plan users